### ISSUE TITLE
SA1642 (constructor summary text) cannot be split across lines

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#414]
The following text results in SA1642 being reported because the summary is split across lines:

```csharp
/// <summary>
/// Initializes a new instance of
/// the <see cref="ApiStatus"/> class.
/// </summary>
/// <inheritdoc/>
private ApiStatus(string name)
    : base(name)
{
}
```

**Comment by sharwell** [comment]
I initially assigned @pdelvo since he did a great job with the initial implementation. I'll take a look soon if you don't have time.

**Comment by sharwell** [comment]
I went ahead and fixed this one as well because it was causing many false reports in one of my projects.
